samples/test/support/management-api/upsertPolicyRule.ts

```diff
@@ -65,8 +65,15 @@ const getAccessPolicyActions = (description: string) => {
           'verificationMethod': {
             'factorMode': '1FA',
             'type': 'ASSURANCE',
-            'reauthenticateIn': 'PT1M',
-            'constraints': []
+            'reauthenticateIn': 'PT2H',
+            'constraints': [
```

Comment:
This is required because by default there is a device possession constraint in app sign-in policy which causes password optional tests to fail.